spec/abci++/abci++_app_requirements.md

```diff
@@ -413,18 +423,19 @@ might have a different *CheckTxState* values when they receive it and check thei
 via `CheckTx`.
 Tendermint ignores this value in `ResponseCheckTx`.
 
-From v0.35.x on, there is a `Priority` field in `ResponseCheckTx` that can be
+From v0.37.x on, there is a `Priority` field in `ResponseCheckTx` that can be
```

Comment:
Isn't it from v0.34.x ?
Is it gonna stay on v0.37.x ?

Reply:
For now we do not know what will happen in 0.37 but what might happen if we do get a decision that the priority mempool is out - we keep the code but prevent users from using it for now (at least that was one of the comments from the last meeting). But I can reword all this to 0.34 and then we can change it if need be.

Reply:
Yeah, I would mention "from v0.34.x on", and then if the decision is made to remove it, we can adapt the text.